Issue #3: Added tests for Cryo

fazer1929 opened the pull request:
Depends: https://github.com/jboss-set/cryo/pull/33
Fixes Issue #3
Tests Include:
    1. Check Repository URL
    2. Check determining current branch
    3. Check Fetch PR list
    4. Check future branch setup
    5. Check single PR merge
    6. Check multiple PR merge
    7. Check PR merge with Dependency

baranowb commented:
Is there some setup required? Running CryoTest
Configuring TestNG with: TestNG652Configurator
Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.273 sec - in CryoTest

Results :

Tests run: 0, Failures: 0, Errors: 0, Skipped: 0


Also, this should not be in default package.

fazer1929 committed:
Issue#3: Added tests for Cryo

Tests Include:
    1. Check Repository URL
    2. Check determining current branch
    3. Check Fetch PR list
    4. Check future branch setup
    5. Check single PR merge
    6. Check multiple PR merge
    7. Check PR merge with Dependency
